serverlessworkflow/modules/ROOT/pages/use-cases/advanced-developer-use-cases/getting-started/build-serverless-workflow-quarkus-examples.adoc

```diff
@@ -0,0 +1,64 @@
+= Building images of serverless workflow example applications using Quarkus CLI
```

Comment:
```suggestion
= Building images of SonataFlow example applications using Quarkus CLI
```
WDYT about using SonataFlow instead of serverless workflow? We can mention _serverless workflow_ only in the context of the spec.

Alternatively, we can stick to _workflow service_ to avoid repeating SonataFlow every where.

Reply:
I would personally replace these with `{product_name}` everywhere, but it could get too repetitive.  I understand your point will try to accommodate this into the PR.

Reply:
We can use `{product_name}` every where, but in the same phrase or paragraph we can use _workflow service_ as an alternative to not repeat ourselves that much. wdyt?